docs/workers.md

```diff
@@ -18,14 +18,22 @@ running PostgreSQL.
 
 ## Master/worker communication
 
-The workers communicate with the master process via a Synapse-specific protocol
-called 'replication' (analogous to MySQL- or Postgres-style database
-replication) which feeds a stream of relevant data from the master to the
-workers so they can be kept in sync with the master process and database state.
+The processes communicate with each other via a Synapse-specific protocol called
+'replication' (analogous to MySQL- or Postgres-style database replication) which
+feeds streams of newly written data between processes so they can be kept in
+sync with the database state.
+
+Additionally, processes may make HTTP requests to each other. Typically this is
+used for operations which need to wait for a reply - such as sending an event.
+
+As of Synapse v1.13.0, it is possible to configure Synapse to send replication
```

Comment:
It could be nice to have a snazzy ASCII art diagram here of the processes communicating...

Reply:
+1

Reply:
does https://matrix.sw1v.org/_matrix/media/r0/download/sw1v.org/uhqIFIqONRWcRsKgVvbehsqF help? some of the links have been left out for clarity but I think it might help?

Reply:
Possibly? I'm not really sure who it's aimed at.

Reply:
Am happy to put it at the end of the doc, to be clear.

Reply:
> I'm not really sure who it's aimed at.

well, people trying to set up workers. The idea is that a picture helps people get an overview of how it works.

Reply:
Ok, will add it to the end with a note at the start pointing to it :+1: